.circleci/config.yml

```diff
@@ -25,7 +26,7 @@ jobs:
 
       - run:
           name: Run tests
-          command: yarn test --runInBand
+          command: yarn test
```

Comment:
Do you remember the reason for `--runInBand`?

Reply:
It was running out of memory :)